packages/mobile_app/lib/screens/menu/menu_items.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+import 'package:app/theme/app_colors.dart';
+import 'package:flutter_svg/svg.dart';
+
+GestureDetector menuItemLeadingIcon(
+        {IconData icon,
+        String asset,
+        @required String text,
+        @required Function onTap}) =>
+    GestureDetector(
+      onTap: onTap,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(vertical: 8.0),
+        child: Row(
+          children: [
+            (icon != null)
+                ? Icon(
+                    icon,
+                    color: AppColors.textPrimary,
+                    size: 19.0,
+                  )
+                : (asset != null)
+                    ? SvgPicture.asset(
+                        "assets/icons/$asset.svg",
+                        color: AppColors.textPrimary,
+                        height: 16.0,
+                        width: 16.0,
+                      )
+                    : SizedBox(),
+            SizedBox(width: 17.1),
+            Text(text,
+                style: TextStyle(
+                  color: AppColors.textPrimary,
+                  fontSize: 14,
+                )),
+          ],
+        ),
+      ),
+    );
+
+GestureDetector menuItemFootingIcon(
+        {IconData icon,
```

Comment:
Do we use `icon` and `asset` params anywhere?

Reply:
For some buttons we can use icons, if no icons found - asset from figma.